src/Shared/Common/ParticipantScreeningHelper.cs

```diff
@@ -0,0 +1,62 @@
+using System.Globalization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Functions.Worker.Http;
+
+namespace NHS.ServiceInsights.Common;
+
+public class PaginationHelper
+{
+    private readonly ILogger _logger;
+    public PaginationHelper (ILogger logger)
+    {
+        _logger = logger;
+    }
+
+    public (bool IsValid, int page, int pageSize, DateTime startDate, DateTime endDate) ValidateQuery(HttpRequestData req)
+    {
+        int page = 0;
+        int pageSize = 0;
+        DateTime startDate = default;
+        DateTime endDate = default;
+
+        if (!int.TryParse(req.Query["page"], out page))
+        {
+            _logger.LogError("The page number is invalid.");
+            return (false, 0, 0, default, default);
+        }
+
+        if (!DateTime.TryParse(req.Query["startDate"], CultureInfo.InvariantCulture, out startDate) || !DateTime.TryParse(req.Query["endDate"], CultureInfo.InvariantCulture, out endDate))
+        {
+            _logger.LogError("The startDate or endDate is invalid.");
+            return (false, 0, 0, default, default);
+        }
+
+        if (!int.TryParse(req.Query["pageSize"], out pageSize))
+        {
+            pageSize = 1000;
```

Comment:
Shouldn't this also log an error and return false?

Reply:
How about a log saying "Invalid page size, defaulting to 1000? So that they still get an a page with results, and not an error
```suggestion
            _logger.LogError("The pageSize is invalid. Defaulting to 1000.");
```

Reply:
My concern is that it could be confusing for the caller if they provide an invalid page size or page number and yet still receive an OK response. I don't want that to hold up this PR but I think it's something we should consider and potentially revisit in a separate ticket.